src/main/java/com/epam/emotionalhelp/controller/config/CORSConfig.java

```diff
@@ -0,0 +1,8 @@
+package com.epam.emotionalhelp.controller.config;
+
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
```

Comment:
Do we need to have a configuration here? Let's use [@UtilityClass](https://projectlombok.org/features/experimental/UtilityClass) annotation instead.